src/main/java/uk/gov/hmcts/reform/sendletter/data/LetterRepository.java

```diff
@@ -120,6 +122,20 @@ public int updateIsFailed(UUID id) {
         );
     }
 
+    /**
+     * Retrieve a list of letters which are not failed but still not printed.
+     *
+     * @return list of not printed letters until yesterday's 5pm
+     */
+    public List<NotPrintedLetter> getNotPrintedUntilYesterday() {
+        return jdbcTemplate.query(
+            "SELECT id, message_id, service, type, created_at, sent_to_print_at "
+                + "FROM letters "
+                + "WHERE is_failed = false AND printed_at IS NULL AND sent_to_print_at IS NOT NULL"
+                + "  AND sent_to_print_at < (CURRENT_DATE - integer '1' + time '17:00')",
```

Comment:
it would be nice to have each `AND` in a new line imho

Reply:
there are a lot nice things. new line ANDs are one of them